Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use Node.js 20 as the base image
+FROM node:20
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy package.json and package-lock.json to the container
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy the rest of the application code to the container
+COPY . .
+
+# Build the NestJS application
+RUN npm run build
```

Comment:
Update this for production

Reply:
There is no separate command for production.
I have added NODE_ENV Variable, this should make it such that it will build for production.

Reply:
Okay